packages/flutter/test/material/app_bar_test.dart

```diff
@@ -1514,4 +1514,39 @@ void main() {
 
     expect(getMaterialWidget().shape, roundedRectangleBorder);
   });
+
+  testWidgets('SliverAppBar with shape', (WidgetTester tester) async {
+    const RoundedRectangleBorder roundedRectangleBorder = RoundedRectangleBorder(
+      borderRadius: BorderRadius.all(Radius.circular(15.0)),
+    );
+    await tester.pumpWidget(
+      MaterialApp(
+        home: CustomScrollView(
+          slivers: const <Widget>[
+            SliverAppBar(
+              leading: Text('L'),
+              title: Text('No Scaffold'),
+              shape: roundedRectangleBorder,
+              actions: <Widget>[Text('A1'), Text('A2')],
+            ),
+          ],
+        ),
+      ),
+    );
+
+    final Finder sliverAppBarFinder = find.byType(SliverAppBar);
+
```

Comment:
Remove newline to keep formatting consistent (there is no space between line 1545 and 1546 below)

Reply:
Could you also remove line 1503 as well?